recipe/meta.yaml

```
@@ -26,37 +26,39 @@ requirements:
   build:
     # pythran code needs clang-cl on windows
     - clang                 # [win]
+    - lld     # [win]
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
     - llvm-openmp >=4.0.1   # [osx]
     - m2-patch              # [win]
     - patch                 # [not win]
   host:
     - python
-    - cython >=0.29.24,<3.0
+    - cython >=0.29.24
     - llvm-openmp >=4.0.01  # [osx]
-    - numpy   1.19  # [py<310]
-    - numpy   1.21  # [py==310]
-    - numpy   1.23  # [py>=311]
+    - numpy {{ numpy }}
```

```suggestion
    - numpy 1.21  # [py<311]
    - numpy 1.23  # [py>=311]
```
https://github.com/scikit-image/scikit-image/blob/v0.20.0/pyproject.toml#L39
(Otherwise it will pick numpy 1.16 for 3.8 and 3.9)